courses/free-courses-en.md

```diff
@@ -319,6 +319,7 @@
 * [GitHowTo](https://githowto.com)
 * [How to Use Git and GitHub](https://www.udacity.com/course/how-to-use-git-and-github--ud775) (Udacity)
 * [Introduction to Git and GitHub](https://www.coursera.org/learn/introduction-git-github) - Google (Coursera)
+* [Learn Version Control system with GIT](https://geeksgod.com/udemy-free-course/git-for-beginners/)
```

Comment:
Following guidelines:
- tracking sites / parameters are not accepted. The right link is https://www.eduonix.com/git-for-beginners and then, the title `GIT for beginners`
- should give creator attributions: `Maksym Rudnyi`
- should anotate platform: `Eduonix Learning Solutions`

So, try to fix it with:

```markdown
* [GIT for beginners](https://geeksgod.com/udemy-free-course/git-for-beginners/) - Maksym Rudnyi (Eduonix Learning Solutions)
```
And then, reorder to satisfy the list alphabetization rule.